exercises/complex-numbers/README.md

```diff
@@ -0,0 +1,37 @@
+# Complex Numbers
+
+A complex number is a number in the form `a + b * i` where `a` and `b` are real and `i` satisfies `i^2 = -1`.
+
+Assume the programming language you are using does not have an implementation of complex numbers.
```

Comment:
This doesn't actually say what we're supposed to do.
(This may be a problem present in the problem-specifications repository.)

Reply:
You are right.
I looked at Java and Python implementations and they have the same message copied from problem-specifications.